### ISSUE TITLE
riff system install segfaults

### ISSUE THREAD

**Issue by scothis** [projectriff/riff#821]
The `riff system install` and `uninstall` commands are segfaulting after #788

```
$ riff system install
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1c36e20]

goroutine 1 [running]:
github.com/projectriff/riff/cmd/commands.SystemInstall.func1(0xc000316280, 0x2b255e0, 0x0, 0x0, 0x0, 0x0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/cmd/commands/system.go:56 +0x30
github.com/projectriff/riff/cmd/commands.CreateAndWireRootCommand.func2.1(0xc000316280, 0x2b255e0, 0x0, 0x0, 0x0, 0x0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/cmd/commands/wiring.go:180 +0x58
github.com/projectriff/riff/vendor/github.com/spf13/cobra.(*Command).execute(0xc000316280, 0x2b255e0, 0x0, 0x0, 0xc000316280, 0x2b255e0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/vendor/github.com/spf13/cobra/command.go:762 +0x473
github.com/projectriff/riff/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc00047f180, 0x2, 0xc0004203c0, 0xc0004203a0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
main.main()
        /Users/scothis/Development/go/src/github.com/projectriff/riff/main.go:31 +0x2b
```

**Comment by fbiville** [comment]
Same thing with `riff subscription create`, I'll open another issue.

**Comment by glyn** [comment]
Same thing with `riff image relocate`.
